x/mint/types/minter.go

```diff
@@ -1,9 +1,16 @@
 package types
 
 import (
+	"errors"
+
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+var (
```

Comment:
nit: I think these can be public instead of having to make them private and having them exported

Reply:
I think items that are not meant to be used from the outside of the package, should remain inaccessible.

The `export_test.go` pattern only exports these variables for tests while keeping them inaccessible/unexported by regular packages.